server/methods/bulkActions.js

```diff
@@ -15,9 +15,9 @@ Meteor.methods({
   },
   'advancedFulfillment/unshipSelectedOrders': function (orderIds) {
     check(orderIds, [String]);
-    // if (!ReactionCore.hasPermission('reaction-advanced-fulfillment')) {
-    //   throw new Meteor.Error(403, 'Access Denied');
-    // }
+    if (!ReactionCore.hasPermission(advancedFulfillment.server.permissions)) {
+      throw new Meteor.Error(403, 'Access Denied');
+    }
     this.unblock();
```

Comment:
@spencern  is hasPermission a Reaction Built function or is that part of accounts or some package?